pgbot/__init__.py

```diff
@@ -73,6 +75,12 @@ async def _init():
                 if channel.id == value:
                     common.entry_channels[key] = channel
 
+    async with db.DiscordDB("blacklist") as db_obj:  # disable blacklisted commands
+        for cmd_qualname in db_obj.get([]):
+            cmd = common.bot.get_command(cmd_qualname)
+            if cmd is not None:
+                cmd.update(enabled=False)
```

Comment:
If this is done in the init function, we cannot modify the blacklist at runtime and have it effect immediately without restarting the entire bot... so best to do this elsewhere IG

Reply:
Well, but I've tested the code in `[**blacklist_cmd**](https://github.com/PygameCommunityDiscord/PygameCommunityBot/blob/399d8af82b59ad1afcbcc7d2e0957c3ce9fa1499/pgbot/commands/admin/__init__.py#L196)` and `[**whitelist_cmd**](https://github.com/PygameCommunityDiscord/PygameCommunityBot/blob/399d8af82b59ad1afcbcc7d2e0957c3ce9fa1499/pgbot/commands/admin/__init__.py#L167)` and that is not a problem at all.

Reply:
Where else would you suggest, this is where the command cogs are being registered, hence why the `Command` objects are updated here.

Reply:
> we cannot modify the blacklist at runtime and have it effect immediately
You can update `Command` objects at runtime ;)

Reply:
Seems like this is resolved now.